_cucumber/features/support/hooks.rb

```diff
@@ -59,6 +59,11 @@
 end
 
 def user_logout
-  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout?redirect_uri=#{@redirect_url}%3Fredirect_fragment%3D!")
+  if Capybara.app_host.include?('stage') || Capybara.app_host.include?('docker')
```

Comment:
This was also part of a previous PR, I just added the production URL in this PR.